agent/api/eni/eni.go

```diff
@@ -90,7 +92,8 @@ const (
 
 var (
 	// netInterfaces is the Interfaces() method of net package.
```

Comment:
nit: Do you want to update/remove this comment?

Reply:
Good call removed.